action.yaml

```diff
@@ -3,4 +3,4 @@ description: "Sonarqube"
 
 runs:
   using: "docker"
-  image: docker://cicd-oci-release-orl.repo.orl.eng.hitachivantara.com/lumada-common-services/gh-composite-actions:sonar-action
+  image: docker://one.hitachivantara.com/devops-docker-release/lumada-common-services/gh-composite-actions:sonar-action.27
```

Comment:
This could be `docker` instead of `devops-docker-release`

Reply:
updated.